platform/platform_win.cpp

```diff
@@ -192,3 +209,26 @@ bool Platform::GetFileSizeByFullPath(string const & filePath, uint64_t & size)
   }
   return false;
 }
+
+time_t Platform::GetFileCreationTime(std::string const & path)
```

Comment:
Please try to reuse (if possible) the internal implementation and also add the recently introduced `GetFileModificationTime` method.

Reply:
What implementation are you referring to?

Reply:
I meant share the code between Creation and Modification functions.